tests/UnitTests.hs

```diff
@@ -198,6 +198,14 @@ utcTimeGood = do
   assertEqual "utctime" (parseWithRead "%FT%H:%MZ" ts10) t10
   assertEqual "utctime" (parseWithRead "%F %H:%MZ" ts11) t11
   assertEqual "utctime" (parseWithRead "%FT%T%QZ" "2015-01-01T14:30:00Z") t12
+  
+  -- leap seconds are included correctly
+  let ts13 = "2015-08-23T23:59:60.128+00" :: LT.Text
```

Comment:
is `23:59:61` however rejected?

Reply:
Should be -- I added a negative test case.